### PR TITLE
Update variables.tf

### PR DESCRIPTION
**Pull request by acolleen**
### Description

I've completed a content review and made edits to the `variables.tf` file as part of AHA Epic DAF-303. This review is related to the following issues:

https://github.ibm.com/GoldenEye/issues/issues/7807
https://github.ibm.com/Bluemix/docs/issues/10672

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by shemau — APPROVED** [review]
LGTM

**Comment by shemau** [comment]
No release required. This may fail and need a refresh, since README.md contains variable descriptions.

If it does, I will take a look locally. The branch will need checking out, `pre-commit  run --all-files` run against it and any changes committed.

File: solutions/apps/variables.tf [hunk 1]
```diff
@@ -156,27 +156,27 @@ variable "scale_memory_limit" {
...
 variable "scale_request_timeout" {
-  description = "The amount of time in seconds that is allowed for a running app to respond to a request."
+  description = "The amount of time in seconds during which a running app can respond to a request."
   type        = number
   default     = 300
 }
 
 variable "config_maps" {
-  description = "A map of code engine config maps to be created. Example: { configmap_name: {data: {key_1: 'value_1' }}}"
+  description = "A map of the IBM Cloud Code Engine configmaps to create. For example, `{ configmap_name: {data: {key_1: 'value_1' }}}`.
```
Comment (shemau) [inline solutions/apps/variables.tf:171]: Lost double quote to terminate description string.

File: solutions/apps/variables.tf [hunk 2]
```diff
@@ -156,27 +156,27 @@ variable "scale_memory_limit" {
...
   type = map(object({
     data = map(string)
   }))
   default = {}
 }
 
 variable "secrets" {
-  description = "A map of code engine secrets to be created. Example: { secret_name: {format: 'generic', data: {key_1: 'value_1' }}}"
+  description = "A map of the IBM Cloud Code Engine secrets to create. For example, `{ secret_name: {format: 'generic', data: {key_1: 'value_1' }}}`.
```
Comment (shemau) [inline solutions/apps/variables.tf:179]: Lost double quote to terminate description string.

File: solutions/apps/variables.tf [hunk 3]
```diff
@@ -197,7 +197,7 @@ variable "secrets" {
 }
 
 variable "domain_mappings" {
-  description = "A map of code engine domain mappings to be created. Example: { domain_mapping_name: {tls_secret: 'tls_secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}" # pragma: allowlist secret
+  description = "A map of the IBM Cloud Code Engine domain mappings to create. For example, `{ domain_mapping_name: {tls_secret: 'tls_secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}`. # pragma: allowlist secret
```
Comment (shemau) [inline solutions/apps/variables.tf:200]: Lost double quote to terminate description string. (before #pragma)

File: solutions/apps/variables.tf [hunk 4]
```diff
@@ -209,7 +209,7 @@ variable "domain_mappings" {
 }
 
 variable "bindings" {
-  description = "A map of code engine bindings to be created. Example: { 'PREFIX': {secret_name: 'secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}"
+  description = "A map of the IBM Cloud Code Engine bindings to create. For example, `{ 'PREFIX': {secret_name: 'secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}`.
```
Comment (shemau) [inline solutions/apps/variables.tf:212]: Lost double quote to terminate description string.

**Review by shemau — CHANGES_REQUESTED** [review]
I wonder if some of the changes should also be made to (../../)variables.tf, so that they show up in the README.md for the module as well as on the inputs that will be visible in the catalog module/schematics/projects interfaces.

File: solutions/apps/variables.tf (hunk 1 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: I've fixed this issue in the PR.

File: solutions/apps/variables.tf (hunk 2 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: I've fixed this issue in the PR.

File: solutions/apps/variables.tf (hunk 3 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: I've fixed this issue in the PR.

File: solutions/apps/variables.tf (hunk 4 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: I've fixed this issue in the PR.

**Comment by ocofaigh** [comment]
NOTE: Lets generate a release for this PR so all the Prs I just merged with doc updates are part of a version in the catalog.

File: solutions/apps/variables.tf [hunk 5]
```diff
@@ -156,27 +156,27 @@ variable "scale_memory_limit" {
...
 variable "scale_request_timeout" {
-  description = "The amount of time in seconds that is allowed for a running app to respond to a request."
+  description = "The amount of time in seconds during which a running app can respond to a request."
   type        = number
   default     = 300
 }
 
 variable "config_maps" {
-  description = "A map of code engine config maps to be created. Example: { configmap_name: {data: {key_1: 'value_1' }}}"
+  description = "A map of the IBM Cloud Code Engine configmaps to create. For example, `{ configmap_name: {data: {key_1: 'value_1' }}}`".
```
Comment (akocbek) [inline solutions/apps/variables.tf:171]: dot `.` should inside double quotation mark
```
}}}`."
```

File: solutions/apps/variables.tf [hunk 6]
```diff
@@ -156,27 +156,27 @@ variable "scale_memory_limit" {
...
   type = map(object({
     data = map(string)
   }))
   default = {}
 }
 
 variable "secrets" {
-  description = "A map of code engine secrets to be created. Example: { secret_name: {format: 'generic', data: {key_1: 'value_1' }}}"
+  description = "A map of the IBM Cloud Code Engine secrets to create. For example, `{ secret_name: {format: 'generic', data: {key_1: 'value_1' }}}`".
```
Comment (akocbek) [inline solutions/apps/variables.tf:179]: dot `.` should inside double quotation mark
```
}}}`."
```

File: solutions/apps/variables.tf [hunk 7]
```diff
@@ -197,7 +197,7 @@ variable "secrets" {
 }
 
 variable "domain_mappings" {
-  description = "A map of code engine domain mappings to be created. Example: { domain_mapping_name: {tls_secret: 'tls_secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}" # pragma: allowlist secret
+  description = "A map of the IBM Cloud Code Engine domain mappings to create. For example, `{ domain_mapping_name: {tls_secret: 'tls_secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}`". # pragma: allowlist secret
```
Comment (akocbek) [inline solutions/apps/variables.tf:200]: dot `.` should inside double quotation mark
```
}}}`."
```

File: solutions/apps/variables.tf [hunk 8]
```diff
@@ -209,7 +209,7 @@ variable "domain_mappings" {
 }
 
 variable "bindings" {
-  description = "A map of code engine bindings to be created. Example: { 'PREFIX': {secret_name: 'secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}"
+  description = "A map of the IBM Cloud Code Engine bindings to create. For example, `{ 'PREFIX': {secret_name: 'secret_name', components: [{ name : 'app_name', resource_type: 'app_v2'}]}}`".
```
Comment (akocbek) [inline solutions/apps/variables.tf:212]: dot `.` should inside double quotation mark
```
}}}`."
```

**Review by akocbek — CHANGES_REQUESTED** [review]
small typos

File: solutions/apps/variables.tf (hunk 5 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: Fixed.

File: solutions/apps/variables.tf (hunk 6 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: Fixed.

File: solutions/apps/variables.tf (hunk 7 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: Fixed.

File: solutions/apps/variables.tf (hunk 8 above)
Comment (acolleen) [reply solutions/apps/variables.tf]: Fixed.

**Review by akocbek — APPROVED** [review]

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**